src/sagas.js

```diff
@@ -664,31 +669,54 @@ function* fetchAllNetworks () {
   }
 }
 
-function* schedulerWithFixedDelay (action) {
-  logDebug('Starting schedulerWithFixedDelay() scheduler')
+function* delayedRemoveActiveRequest ({ payload: requestId }) {
+  yield delay(500)
+  yield put(removeActiveRequest(requestId))
+}
 
-  // TODO: do we need to stop the loop? Consider takeLatest in the rootSaga 'restarts' the loop if needed
-  while (true) {
-    yield delay(AppConfiguration.schedulerFixedDelayInSeconds * 1000)
-    logDebug(`schedulerWithFixedDelay() event after delay of ${AppConfiguration.schedulerFixedDelayInSeconds} seconds`)
+function* startSchedulerWithFixedDelay (action) {
+  // if a scheduler is already running, stop it
+  if (Selectors.isSchedulerEnabled()) {
```

Comment:
Here must be `if (!Selectors.isSchedulerEnabled()) {`, no?

Reply:
No.

It is setup to have only 1 scheduler running at a time.  When starting, if the scheduler is already enabled, stop it.

Reply:
I was able to take out the `isSchedulerEnabled` selector and reducer etc.  It wasn't needed.